appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/reject/reject.controller.js

```diff
@@ -189,3 +197,19 @@ export const renderCheckYourAnswers = async (request, response) => {
 		pageContent
 	});
 };
+
+/**
+ * @param {import('@pins/express/types/express.js').Request} request
+ * @param {import('@pins/express/types/express.js').RenderedResponse<any, any, Number>} response
+ * @param {() => any} next
+ * */
+export async function initialiseSession(request, response, next) {
```

Comment:
Personally I'd've filed this under middleware but you do you

Reply:
👍 Good idea